### ISSUE TITLE
Displaying reservation information in Scan application 

### ISSUE THREAD

**Issue by michalzubkowicz** [alfio-event/alf.io#1326]
**Is your feature request related to a problem? Please describe.**
I want to implement displaying of information about reservation in the Scan application. For example - people at conference will get different badges when they have the ie. meals included. Person responsible for scanning should see additional information to select correct badge. 

**Describe the solution you'd like**
In the scan.common.ts in scanning app there is a model TicketFieldValueForAdditionalService
https://github.com/alfio-event/Alf.ioBackoffice/blob/master/src/app/shared/scan/scan-common.ts

I think this model is intended to provide such information. Should I use this model and only implement displaying information in Scan application (currently this information is unused). 

**Describe alternatives you've considered**
I was thinking also about providing additional field in TicketAndCheckInResult to provide such information and displaying it. 

**Additional context**
Displaying additional information about ticket should be configurable or I should implement always showing ticket details ?

**Comment by cbellone** [comment]
Hi,

we already provide information for:

**different categories**
(edit category -> badge color) 
for differentiating specific type of attendees (i.e. speakers at a conference). 
If some attendees have meal included and other don't, you can create two different categories and then give them different colors

**additional items and linked info**
If you want to _sell_ meals as an additional option, then at check-in you should see who bought what. 
This is valid for all merchandise / additional options. For example you can sell t-shirt and then ask which size. All will reported in the check-in app upon successful scan

would that cover your use case?

**Comment by michalzubkowicz** [comment]
Hi
@cbellone , thanks for information, you saved me a lot of digging. Yes it covers! There is somewhere whole process described?

In example, when I create reservation under reservation link there is no "additional options" displayed, only after I click "buy additional tickets". I don't understand why it happens this way.

**Comment by michalzubkowicz** [comment]
I was digging in the source code and I've found that additional options are showing only before creating reservation. After you click reservation link reservation is created already so you don't have to create additional options. 

It could be nice to have somewhere this process explained.
